app/build.gradle.kts

```diff
@@ -96,8 +90,6 @@ dependencies {
 }
 
 ksp {
-    // Anvil-KSP
-    arg("anvil-ksp-extraContributingAnnotations", "com.slack.circuit.codegen.annotations.CircuitInject")
-    // kotlin-inject-anvil (requires 0.0.3+)
-    arg("kotlin-inject-anvil-contributing-annotations", "com.slack.circuit.codegen.annotations.CircuitInject")
+    // Circuit-KSP for Metro
+    arg("circuit.codegen.mode", "metro")
```

Comment:
Not sure why is this needed - https://github.com/ZacSweers/metro/blob/main/samples/circuit-app/build.gradle.kts#L16

I didn't see it in the doc.